app/auth/controllers.test.js

```diff
@@ -1,4 +1,5 @@
 const controllers = require('./controllers')
+const { AUTH_PROVIDERS, DEFAULT_AUTH_PROVIDER } = require('../../config')
```

Comment:
As we spoke about yesterday, might be better to mock these values using proxyquire rather than relying on the actual values set within the config module.